pom.xml

```diff
@@ -20,13 +20,13 @@
   <parent>
     <groupId>org.sonatype.nexus.plugins</groupId>
     <artifactId>nexus-plugins</artifactId>
-    <version>3.16.1-02</version>
+    <version>3.17.0-01</version>
   </parent>
 
   <artifactId>nexus-blobstore-google-cloud</artifactId>
   <name>${project.groupId}:${project.artifactId}</name>
   <inceptionYear>2017</inceptionYear>
-  <version>0.6.1</version>
+  <version>0.7.0</version>
```

Comment:
@DarthHater With the new CirceCI process - is it correct for me to skip the -SNAPSHOT suffix?